docker-compose.yml

```diff
@@ -1,4 +1,4 @@
-version: '3'
+dersion: '3'
```

Comment:
is this deliberate?

Reply:
oh. weird. you'd think that would have broken something lol?